audioipc/src/lib.rs

```diff
@@ -200,13 +200,9 @@ pub(crate) unsafe fn close_target_handle(
 
 #[cfg(windows)]
 pub fn server_platform_init() {
-    use winapi::shared::winerror;
-    use winapi::um::combaseapi;
-    use winapi::um::objbase;
-
     unsafe {
-        let r = combaseapi::CoInitializeEx(std::ptr::null_mut(), objbase::COINIT_MULTITHREADED);
-        assert!(winerror::SUCCEEDED(r));
+        let r = CoInitializeEx(std::ptr::null_mut(), COINIT_MULTITHREADED);
+        assert!(r == S_OK || r == S_FALSE);
```

Comment:
The return value changes from success to (ok or false). Is there anything making it not always `SUCCEEDED` anymore?

Reply:
`SUCCEEDED` treated both `S_OK` and `S_FALSE` as success so the only change here is that `windows-sys` doesn't provide an equivalent macro for this check.